nilearn/datasets/func.py

```diff
@@ -1276,3 +1276,174 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
         X, y, mask_img = _load_mixed_gambles(map(nibabel.load, data.zmaps))
         data.zmaps, data.gain, data.mask_img = X, y, mask_img
     return data
+
+
+def fetch_megatrawls_netmats(data_dir=None, choice_dimensionality=None,
+                             choice_timeseries=None,
+                             resume=True, verbose=1):
+    """Downloads and fetches Network Matrices data from MegaTrawls release in HCP.
+
+    This data can be used to predict relationships between imaging data (functional
+    connectivity) and non-imaging behavioural measures such as age, sex, education, etc.
+    The network matrices are estimated between 461 functional connectivity subjects.
```

Comment:
I'd rather put "The network matrices are estimated from functional connectivity datasets of 461 subjects. "